examples/quickstarts/nginx.json

```diff
@@ -0,0 +1 @@
+404: Not Found
```

Comment:
that clearly didn't work :)

Reply:
Yes, I found out right now :)
Will you copy `nginx-ex` to `openshift` or should I change it to `sclorg`?

Reply:
since you guys own it, i think you should put it in sclorg.

Reply:
Does it mean we'll copy also the other sample apps that we're to maintain to `sclorg`?

Reply:
that's probably a discussion we should have, anyway.

Reply:
@hhorak, WDYT?

Reply:
fyi we're ok with it from our side.